src/org/javaday/java8/misused/Permission.java

```diff
@@ -7,12 +7,8 @@
 // ============================================================================
 //  Copyright(c) 2016 XP Injection, Ukraine
 // ============================================================================
-package com.xpinjection.misuses;
+package org.javaday.java8.misused;
```

Comment:
Don't like an idea of changing packages to single conference. The talk is usually demonstrated several times and such naming conventions always look strange for participants.